src/Glpi/Application/ErrorHandler.php

```diff
@@ -393,23 +396,11 @@ public function handleException(\Throwable $exception, bool $quiet = false): voi
     {
         $this->exit_code = 255;
 
-        $error_type = sprintf(
-            'Uncaught Exception %s',
-            get_class($exception)
-        );
-        $error_description = sprintf(
-            '%s in %s at line %s',
-            $exception->getMessage(),
-            $exception->getFile(),
-            $exception->getLine()
-        );
-        $error_trace = $this->getTraceAsString($exception->getTrace());
-
-        $log_level = self::ERROR_LEVEL_MAP[E_ERROR];
+        $this->logException($exception);
 
-        $this->logErrorMessage($error_type, $error_description, $error_trace, $log_level);
-        if (!$quiet) {
-            $this->outputDebugMessage($error_type, $error_description, $log_level);
+        if (!$quiet && isCommandLine()) {
+            $log_level = self::ERROR_LEVEL_MAP[$exception->getCode()] ?? 255;
+            $this->outputDebugMessage($exception->getCode(), $exception->getMessage(), $log_level);
```

Comment:
The full stack trace will not be present anymore in logs. This change should be reverted.

Reply:
I confirm that the stack trace is available in all possible cases of errors (I tested them) except for memory limit where we only have the file and line

Reply:
My bad, I misread the change. I mixed output and logs in my mind.

Reply:
But the previous output was more precise as it was containing the exception class, the file and the line..

Reply:
Here's an example of the logs when an exception is triggered:

```
[2024-08-01 13:58:31] glpiphplog.CRITICAL:   *** Uncaught Exception RuntimeException: Exception triggered in /var/www/glpi/front/error_exception.php at line 35
  Backtrace :
  ...Glpi/Controller/LegacyFileLoadController.php:57 require()
  vendor/symfony/http-kernel/HttpKernel.php:101      Glpi\Controller\LegacyFileLoadController->Glpi\Controller\{closure}()
  ...ymfony/http-foundation/StreamedResponse.php:106 Symfony\Component\HttpKernel\HttpKernel::Symfony\Component\HttpKernel\{closure}()
  vendor/symfony/http-foundation/Response.php:423    Symfony\Component\HttpFoundation\StreamedResponse->sendContent()
  public/index.php:58                                Symfony\Component\HttpFoundation\Response->send()
```

Reply:
The logs seems OK, but the output message that will be displayed in the console should not have changed. The text was more precise.
The `testHandleException` should not have changed too.

Reply:
The screenshots above are for the "non-dev" setups.

In dev, it provides the stack trace like this: 
![image](https://github.com/user-attachments/assets/733cb840-e9ff-4a1a-bb6a-d4f63969f876)

Reply:
I am talking about the console output that changed, according to this change in tests:
![image](https://github.com/user-attachments/assets/d255ae25-709c-4854-9118-de1029ea9064)